### PR TITLE
Bug 1168962 - Expand/Collapse Browser Chrome on Navigation r=etienne

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0c2014c93eff96bfbb791921f8a5b738bee3e504

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a6b3c83d491c7ab20104de362589decc1b0ad05f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=318973c3183ffffcecdeb15651d819cee12b4085

File: apps/system/test/unit/app_chrome_test.js [hunk 1]
```diff
@@ -607,20 +607,42 @@ suite('system/AppChrome', function() {
...
+      });
+      fakeSearchApp.chrome.pinned = true;
+      fakeSearchApp.chrome.scrollable = true;
+      fakeSearchApp.chrome.url = 'http://aaa.com';
+      app = new AppWindow(fakeSearchApp);
+      this.sinon.stub(app, 'isBrowser').returns(true);
+      chrome = new AppChrome(app);
       chrome.handleLocationChange();
       assert.isFalse(chrome.pinned);
-      assert.isTrue(app.element.classList.contains('collapsible'));
```
Comment (etiennesegonzac) [inline apps/system/test/unit/app_chrome_test.js]: nit: I'd keep an `assert.isTrue(app.element.classList.contains('collapsible'));`

File: apps/system/test/unit/app_chrome_test.js [hunk 2]
```diff
@@ -607,20 +607,42 @@ suite('system/AppChrome', function() {
...
       fakeSearchApp.chrome.pinned = true;
       fakeSearchApp.chrome.scrollable = true;
       fakeSearchApp.chrome.url = 'http://aaa.com';
-      var app = new AppWindow(fakeSearchApp);
+      app = new AppWindow(fakeSearchApp);
       this.sinon.stub(app, 'isBrowser').returns(true);
-      var chrome = new AppChrome(app);
+      chrome = new AppChrome(app);
       chrome.handleLocationChange();
       assert.isTrue(chrome.pinned);
```
Comment (etiennesegonzac) [inline apps/system/test/unit/app_chrome_test.js]: and add `assert.isFalse(app.element.classList.contains('collapsible'));`

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=28c29b9ea4b25669a12454fdb1a95a9ec9a14165

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ebc318cc840b13a75d3fb353c11b5c5c8aeed508

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c23fc6796f69dfde5dee0ae08a0721aee48411ea

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bbb5ba2a74c56218d68b79f9cc0d65d7f3eebd9d

**Pull request merged**